dev-tex/biber/files/biber-2.16-disable-64bit-only-tests.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/t/dateformats.t b/t/dateformats.t
+index 9c179f06..73c87de1 100644
+--- a/t/dateformats.t
++++ b/t/dateformats.t
+@@ -4,7 +4,7 @@ use warnings;
+ use utf8;
+ no warnings 'utf8';
+ 
+-use Test::More tests => 56;
++use Test::More tests => 48;
+ use Test::Differences;
+ unified_diff;
+ 
+@@ -880,14 +880,14 @@ $biber->prepare;
+ $out = $biber->get_output_obj;
+ 
+ # Test negative dates and eras
+-eq_or_diff($out->get_output_entry('era1', $main), $era1, 'Date meta information - 1');
+-eq_or_diff($out->get_output_entry('era2', $main), $era2, 'Date meta information - 2');
+-eq_or_diff($out->get_output_entry('era3', $main), $era3, 'Date meta information - 3');
+-eq_or_diff($out->get_output_entry('era4', $main), $era4, 'Date meta information - 4');
++#eq_or_diff($out->get_output_entry('era1', $main), $era1, 'Date meta information - 1');
```

Comment:
We usually don't commit patches with comments, we just remove the lines. The patch shows us which lines have been removed so information is lost. I need to update [this](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto) accordingly